### ISSUE TITLE
`dist` folder disappeared in npm bundle (v0.4.0)

### ISSUE THREAD

**Issue by s-panferov** [cbowdon/TsMonad#10]
`ls node_modules/tsmonad`:

```
➜  git:(master) ✗ ls node_modules/tsmonad
LICENSE-MIT  README.md    maybe.ts     monad.ts     package.json tsmonad.ts   writer.ts
```

**Comment by cbowdon** [comment]
D'oh. Thank you... fix pending ASAP.

**Comment by cbowdon** [comment]
Fixed in v0.4.1 (5ea3691c5675b1c9c37702012f885a3907b1c430) - let me know if you still have any issues. It was because I used `npm publish` instead of `npm publish <folder path>`.
